src/deterministic.jl

```diff
@@ -91,25 +79,25 @@ function Deterministic(
         try
             data[k] = Float64[i for i in v]
         catch e
-            @error("The forecast data provided $(second(eltype(input_data))) can't be converted to Vector{Float64}")
+            @error("The forecast data provided $(eltype(input_data)) can't be converted to Vector{Float64}")
```

Comment:
@jd-lara What did you intend with `second`?  It is not a function.

Reply:
This stale code from the time we iterated over pairs. 

```
second(a=>b) = b
```